src/single-spa.js

```diff
@@ -1,556 +1,18 @@
-import CustomEvent from 'custom-event';
-import { handleChildAppError } from './single-spa-child-app-error.js';
-import { ensureJQuerySupport } from './jquery-support.js';
-import { parseUri } from './single-spa.helpers.js';
-
-const window = typeof window !== 'undefined' ? window : (typeof global !== 'undefined' ? global : self);
-
-const originalAddEventListener = window.addEventListener;
-const originalRemoveEventListener = window.removeEventListener;
-
-// App statuses
-const NOT_BOOTSTRAPPED = 'NOT_BOOTSTRAPPED',
-	LOADING_SOURCE_CODE = 'LOADING_SOURCE_CODE',
-	BOOTSTRAPPING = 'BOOTSTRAPPING',
-	NOT_MOUNTED = 'NOT_MOUNTED',
-	MOUNTING = 'MOUNTING',
-	MOUNTED = 'MOUNTED',
-	UNMOUNTING = 'UNMOUNTING',
-	SKIP_BECAUSE_BROKEN = 'SKIP_BECAUSE_BROKEN';
-
-// Constants that don't change no matter what
-export const routingEventsListeningTo = ['hashchange', 'popstate'];
-
-// Things that need to be reset with the init function;
-let Loader, childApps, globalTimeoutConfig, peopleWaitingOnAppChange, appChangeUnderway, capturedEventListeners, isPaused;
-
+export { start } from './start.js';
```

Comment:
This is the entry file and simply exports everything that needs to be exposed in the API. This api should be fully documented in the single-spa-api.md file.